ast/src/parsed/mod.rs

```diff
@@ -633,6 +635,71 @@ pub enum BinaryOperator {
     Greater,
 }
 
+#[derive(Debug, PartialEq, Eq)]
+pub enum BinaryOperatorAssociativity {
+    Left,
+    Right,
+    RequireParentheses,
+}
+
+const UNARY_EXPRESSION_PRIORITY_OFFSET: ExpressionPrecedence = 0; // All unary operators have the same precedence.
```

Comment:
Not sure this is a good idea, I think it distracts. Shouldn't we just stick to the table in the rust documentation?